### ISSUE TITLE
Need a MacOSX Release

### ISSUE THREAD

**Issue by tumugide** [subhra74/xdm-experimental-binaries#17]
Dear team it is highly needed for the MacOS users to also have access to the wonderful xdm we all love, if it's possible you can let us know at least how we can help to make that happen.
Thanks

**Comment by mokun** [comment]
Yes agree. Can someone by any chance compile the 8.0.12 beta for the Apple Silicon (M1/M2) ? Much appreciate !

**Comment by mokun** [comment]
@subhra74 Yes agree. Can someone by any chance compile the 8.0.12 beta for the Apple Silicon (M1/M2) ? Much appreciate !

**Comment by subhra74** [comment]
I think homebrew package for mac can be created.. based on Gtk3. I am thinking about creating native Mac ui which requires quite lot of work, also not sure how many mac users use XDM

**Comment by mja0342** [comment]
@subhra74 there are a lot of mac users that uses XDM. that is great one

**Comment by akindoju** [comment]
Hi, please is there any update on this, much needed. Thanks.

**Comment by rony747** [comment]
Please consider a mac os release. this is the best downloader. Thanks for the great work.
